pkg/sysctl/sysctl.go

```diff
@@ -0,0 +1,95 @@
+package sysctl
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+const (
+	// sysctlBase is the root directory for sysctl values in the proc filesystem
+	sysctlBase = "/proc/sys"
+	// NFConntrackMax defines how many connection track entries can be active at the same time
+	NFConntrackMax = Sysctl("/net/netfilter/nf_conntrack_max")
+	// NFConntrackMaxSetting defines the maximum settable
+	NFConntrackMaxSetting = 4194304
+
+	// moduleBase is the root directory for module specific settings
+	moduleBase = "/sys/module"
+	// NFConntrackHashSize defines the hashsize of the conntrack module
+	NFConntrackHashSize = Module("/nf_conntrack/parameters/hashsize")
```

Comment:
Public variable is questionable as from the outside this could be overwritten

Reply:
unexported